File: software-engineering-concepts/Refactoring/PHP/PHP Refactoring Best Practices/1Bloaters/3PrimitiveObsession/2IntroduceParameterObject/src/entities/Order.php
<?php 

namespace app\entities;

class Order 
{
  private $items;
  private $voucher;

  public function __construct($items, $voucher)
  {
    $this->items = $items;
    $this->voucher = $voucher;
  }
  public function getItems()
  {
    return $this->items;
  }
  public function setItems($items)
  {
    $this->items = $items;
  }
  public function getVoucher()
  {
    return $this->voucher;
  }
  public function setIVoucher($voucher)
  {
    $this->voucher = $voucher;
  }
}

File: software-engineering-concepts/Refactoring/Java/Java Refactoring Best Practices/Bloaters/3primitive-obsession/BaseProject/src/main/java/com/onlineshop/entities/Order.java
package com.onlineshop.entities;

import com.onlineshop.items.Item;

import java.util.ArrayList;
import java.util.List;

public class Order {


    private List<Item> items;
    private String voucher;

    public Order(){
        items = new ArrayList<>();
    }

    public Order(List<Item> items, String voucher){
        this.items = items;
        this.voucher = voucher;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

    public String getVoucher() {
        return voucher;
    }

    public void setVoucher(String voucher) {
        this.voucher = voucher;
    }
}